dxil_converter.cpp

```diff
@@ -5413,6 +5413,11 @@ bool Converter::Impl::emit_execution_modes_ray_tracing(spv::ExecutionModel model
 	builder.addCapability(spv::CapabilityRayTracingKHR);
 	if (options.ray_tracing_primitive_culling_enabled && shader_analysis.can_require_primitive_culling)
 		builder.addCapability(spv::CapabilityRayTraversalPrimitiveCullingKHR);
+	if (options.opacity_micromap)
```

Comment:
Requires analysis similar to "can require primitive culling".

Reply:
Done, I think, in the same way as primitive culling: conditionally here but still unconditionally for ray query (as long as it was requested by the caller).

I also added CLI switch and renamed the converter option from just `opacity_micromap` to `opacity_micromap_enabled` to be consistent with `ray_tracing_primitive_culling_enabled` but that makes me wonder if the member name of `OptionOpacityMicromap` should stay as `enabled` or if I should rename it to `supported` :thinking: